filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -184,6 +184,8 @@ Options that control how Filebeat deals with log messages that span multiple lin
 
 The following example shows how to configure Filebeat to handle a multiline message where the first line of the message begins with a bracket (`[`).
 
+NOTE: `pattern` is not compatible with format used in `logstash`. See <<regexp-support>> for details or you will face with errors like: `ERR Stop Harvesting. Unexpected encoding line reader error: error parsing regexp: invalid escape sequence: '\d'`
```

Comment:
@alexanderilyin  We've made some changes to our regexp support in Filebeat so that `\d` will be supported in 5.0.
I agree with you that we can be more explicit here, but including the details of the message is too explicit. Can you remove the note here and instead change the description next to pattern (line 213 in your file) to say, "Specifies the regular expression pattern to match. Note that the regexp patterns supported by Filebeat differ somewhat from the patterns supported by Logstash. See <<regexp-support>> for a list of regexp patterns supported by Filebeat." I think it would be also useful to add a similar note to the page about regexp support. You can do that when you resubmit your PR, or I can do that. Thanks for helping us improve the docs!